sympy/core/add.py

```diff
@@ -595,10 +595,13 @@ def _eval_is_nonnegative(self):
         from sympy.core.exprtools import _monotonic_sign
         if not self.is_number:
             c, a = self.as_coeff_Add()
-            if not c.is_zero and a.is_nonnegative:
+            if not c.is_zero:
                 v = _monotonic_sign(a)
```

Comment:
i want to modify ```as_coeff_Add()``` function. i make changes is ```c = -1``` and ```a = p + exp(-1)``` to ```c = -1 + exp(-1)``` and ```a = p```. i mean currently ```as_coeff_Add()``` include ```exp``` in ```a``` part and i change to ```exp``` include in ```c``` part.@asmeurer should i do ?